tests/test_checker/test_noqa.py

```diff
@@ -162,6 +162,7 @@
     'WPS354': 1,
     'WPS355': 1,
     'WPS356': 1,
+    'WPS357': 3,
```

Comment:
This specific violation can be ignored. Please set it to `0` and add a comment `# logically unacceptable.`

Reply:
Fixed. Thanks for reply